scripts/upgrade

```diff
@@ -43,34 +106,46 @@ ynh_secure_remove $final_path/robots.txt
 ynh_secure_remove $final_path/tpl
 ynh_secure_remove $final_path/data
```

Comment:
I think it has been discussed between @julienmalik and @magikcypress .
The rationale is:
* if we come from PrivateBin, we should keep the `data` directory to keep the pastes
* if we upgrade from ZeroBin, we should delete it because you can [maintain compatibility](https://github.com/PrivateBin/PrivateBin/wiki/Configuration#zerobincompatibility) but at a security cost we can't afford.

So, here the comment is not aligned  with the code: you actually delete the `data` directory.
The behaviour should depend on the installed version...

Reply:
Ok, it's appears the data dir was removed since this commit https://github.com/YunoHost-Apps/zerobin_ynh/commit/b1c9f238bff4548f662c1d11d30dce7397fd8d6a.
But, there not distinction between which version you use.

Nevertheless, I admit it should be. Let's keep that for a future enhancement.